# EasyHSSP
A web-based tool for analyzing and visualizing HSSP files
</br></br>
Tremendous development of protein, peptides, and nucleotides sequencing tools leads to high
volume of protein sequencing stored in data bases like Uniprot and NCBI. High volume of protein
sequencing provides the opportunity of implementing different algorithms in the data to suit our
needs; in this regard DSSP algorithm is one of the many algorithms and it delivers DSSP and <b>HSSP</b>
files.
</br></br>
<b>Output of HSSP</b> contains wide variety of information including entropy and amino acids variation
in each amino acids positions in proteins by which protein engineering can be facilitated in terms
of which position is a better candidate to be mutated or which amino acid is an appropriate
candidate to be changed. In a meanwhile, HSSP leaves a gap that this tool address that. The gap
is interpreting HSSP output is hard enough to make protein engineers reluctant to use that.
</br></br>
<b>EasyHSSP</b> makes interpreting the outputs of HSSP much easier. 
</br>
Input of <b>EasyHSSP</b> is the proteins with similar structure chosen
by a user and out puts of that is bar charts showing entropy of each position, probability of each
amino acid in each position, and a new 3D structure of a modified protein in which each type of
amino acid is colored differently, entropy and hydrophilicity of each position is shown.
</br></br>
This type of analyses could be efficient in predicting mutants in a protein sequence; the mutants
can cause many types of diseases by which sometimes an epidemy occurs.
</br></br>
Access to this web-based tool: http://bioinf.modares.ac.ir/software/EasyHSSP/
